devtools/dumper-companion.py

```diff
@@ -905,10 +936,8 @@ def create_macfonts(args: argparse.Namespace) -> int:
     print("Decompressing...", end="")
     datafork.seek(-0x200, 2)
     alt_mdb_loc = datafork.tell()
-    datafork.seek(-(0x200 - 0x12), 2)
-    _, allocation_block_size, first_allocation_block = unpack(
-        ">HI4xH", datafork.read(12)
-    )
+    datafork.seek(-0x1F0, 2)
```

Comment:
Please revert this change as well. It is written this way for better readability

Reply:
Done